change/@fluentui-eslint-plugin-441efdef-6029-4a98-8c60-2019ceecfd84.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
hmm so in normal dependency case this would be a `patch` but because typescript it might be even breaking change ...

lets go with `patch`

https://semver.org/#what-should-i-do-if-i-update-my-own-dependencies-without-changing-the-public-api